mpf/config_spec.yaml

```diff
@@ -582,6 +582,7 @@ hardware_sound_player:
 high_score:
     __valid_in__: mode
     award_slide_display_time: single|ms|4s
+    high_score_reverse_sort: single|bool|False
     categories: omap|str:list|
     defaults: dict|str:list|None
```

Comment:
Would you mind expanding on this a bit more? Like I said elsewhere, I am still new to yaml files, so any guidance you might be able to offer with how this might be structured would be greatly appreciated. Thanks!